src/MobileMapping/Sensor.js

```
@@ -87,6 +87,7 @@ Sensor.prototype.getMatOrientationCapteur = function getMatOrientationCapteur()
         case 2:
             return ori2;
         case 3:
+        default:
```

Not logically equivalent to previous code. Can you elaborate why this works?

---

Definining an arbitrary default value can't hurt, can it? (the alternative would be to raise an error I guess)

---

It depends on how normal the "this.orientation is not 0, 1, 2 or 3" is... I honestly don't know here so I leave that to your fine judgment :-)